### PR TITLE
Changed/added concepts for examples in the article

### PR DESCRIPTION
**Pull request by jrbeaumont**

### CODE REVIEW

File: examples/Celement_with_env_1.hs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+module Concept where
+
+import Tuura.ConceptConcat
+
+-- C-element with environment circuit described using signal-level concepts
+circuit :: (Eq a) => a -> a -> a -> CircuitConcept a 
+circuit a b c = outputRise <> inputFall <> outputFall <>inputRise <> initialState
```
Comment (snowleopard) [inline examples/Celement_with_env_1.hs]: Missing space in `<>inputRise`.

File: examples/Celement_with_env_1.hs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  where
+  outputRise = rise a ~> rise c <> rise b ~> rise c
```
Comment (snowleopard) [inline examples/Celement_with_env_1.hs]: Add two more spaces of indentation here.

File: examples/Celement_with_env_2.hs [hunk 3]
```diff
@@ -0,0 +1,11 @@
+module Concept where
+
+import Tuura.ConceptConcat
+
+-- C-element with environment circuit described using gate-level concepts
+circuit :: (Eq a) => a -> a -> a -> CircuitConcept a
+circuit a b c = cElement a b c <> environment <> initialState
+  where
+  environment = inverter c a <> inverter c b
```
Comment (snowleopard) [inline examples/Celement_with_env_2.hs]: Add two more spaces of indentation here.

**Comment by snowleopard** [comment]
Looks good, just a couple of minor comments.

**Comment by jrbeaumont** [comment]
@snowleopard I should have fixed all of the issues now.

**Comment by snowleopard** [comment]
Merged!

**Pull request merged**